src/Castle.Core.Tests/Castle.Core.Tests-VS2017.csproj

```diff
@@ -1,22 +1,45 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+	
 	<Import Project="..\..\buildscripts\common.props"></Import>
+
+	<PropertyGroup>
+		<TargetFrameworks Condition="'$(IsOSX)'=='true'">netcoreapp1.1</TargetFrameworks>
+		<TargetFrameworks Condition="'$(IsLinux)'=='true'">netcoreapp1.1</TargetFrameworks>
+		<TargetFrameworks Condition="'$(IsWindows)'=='true'">net461;netcoreapp1.1</TargetFrameworks>
+	</PropertyGroup>
```

Comment:
Yes we can, although this only differentiates between Unix and Windows from what I could tell. There might be additional things we would need to do to differentiate between Ubuntu/OSX when we get to enabling matrix builds on TravisCI.